### PR TITLE
ci: add integration tests workflow

### PR DESCRIPTION
**Pull request by badayvedat**

### CODE REVIEW

**Review by isidentical — APPROVED** [review]

**Comment by squat** [comment]
Cool!

File: .github/workflows/integration_tests.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.9"
+
+      - name: Setup python
+        run: python -m venv .venv
```
Comment (squat) [inline .github/workflows/integration_tests.yaml:25]: I think this doesn't do anything because we never activate the virtual environment

Comment (badayvedat) [reply .github/workflows/integration_tests.yaml:25]: yes, you're right! i removed the virtual environment parts in https://github.com/fal-ai/fal/pull/22/commits/61217f3da539d96ab52742c090b48da920919c6c, but it seems like I forgot this section

Comment (badayvedat) [reply .github/workflows/integration_tests.yaml:25]: https://github.com/fal-ai/fal/pull/25

**Pull request merged**